Source/engine/animationinfo.h

```diff
@@ -85,6 +104,10 @@ class AnimationInfo {
 	* @brief Animation Frames that will be adjusted for the skipped Frames/game ticks
 	*/
 	int RelevantFramesForDistributing;
+	/*
+	* @brief Animation Frames that wasn't shown from previous Animation
+	*/
```

Comment:
doxygen
```suggestion
	/**
	 * @brief Animation Frames that wasn't shown from previous Animation
	 */
```

Reply:
Thanks. 🙂
I fixed the two new introduces comments.

There are other comments that contains the same error (introduces by me in previous pr's).
Should I fix them with this pr or do you prefer a separate pr?

btw: git commit --fixup is really handy. Thanks again for the tip 🙂

Reply:
Lets clean up existing issues seperatly for now since this PR is a bout ready